test/master-playlist-controller.test.js

```diff
@@ -825,7 +854,8 @@ async function(assert) {
   assert.equal(MPC.mediaSource.readyState, 'ended', 'Media Source ended');
 });
 
-QUnit.test('does not wait for main loader to finish before calling endOfStream with' +
+// TODO once we have support for audio only with alternate audio
+QUnit.skip('does not wait for main loader to finish before calling endOfStream with' +
```

Comment:
did this used to pass?

Reply:
I think it did on the LHLS branch, but we don't actually have that feature working (AFAIK).

Reply:
🤔

Reply:
we may want to rewrite it so that it fails now (but will pass once we actually have it working)